examples/experimental/Length of Stay demo on MIMIC/example/test_trainer.py

```diff
@@ -0,0 +1,100 @@
+"""module containing functions showing how to use the other functions in the
+    Length of Stay demo on MIMIC folder
+"""
+import os
+import sys
+import json
+import numpy as np
+import logging
+import torch.nn as nn
+from pprint import pformat
+from torch import randint as torch_randint
+from torch import float as torch_float
+from torch.utils.data import TensorDataset
+from torch.utils.data import DataLoader
+from torch.utils.data.sampler import SubsetRandomSampler
+
+print(os.path.dirname(os.path.dirname(__file__)))
+CONFIG_PATH = os.path.join(
+    os.path.dirname(os.path.dirname(__file__)), "metadata", "config_result_paper.json",
+)
+
+
+def test_trainer() -> None:
+    """function showing how to use the train function
+    """
+    sys.path.append(".")
+    os.chdir(os.path.dirname(os.path.dirname(__file__)))
+
+    from pytorch import train
```

Comment:
You can put all the imports at the top :)

Reply:
Indeed it is not a very clean solution for the moment in order to be able to launch this test from anywhere.